data/json/mapgen/veterinarian.json

```diff
@@ -0,0 +1,203 @@
+[
+{
+        "type": "vehicle_group",
+        "id": "VETS",
+        "vehicles": [
+            [ "car", 300 ],
+            [ "pickup", 100 ],
+			[ "suv", 50 ]
+        ]
+    },{
+        "type": "item_group",
+        "id": "vet_softdrug",
+        "items": [
+            ["bandages", 50],
+            ["1st_aid", 10],
+			["saline", 20],
```

Comment:
Your editor should have an option to change tabs to spaces. If it doesn't, you should get a better one - notepad++ is good for it.
Replace all tabs in the file with 4 spaces for each. In case of notepad++, it's (I think) mark everything (ctrl+A) and then edit->whitespace operations->tabs to spaces.

Then make a commit with the new changes and push it.
With online interface, you may need to manually copy the file content once again, but that's just ctrl+c and ctrl+v.

For the future, set the editor to use spaces instead of tabs.